# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/public/GCAPI.h
# Commit: 098569654f17
# Full Hash: 098569654f17fe958cfa7ca7077e4d770044a62e
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2022-10-28 21:22:11
# Regressor Bug: 1797755
# File Overlap Count: 1
# Description:
#   Bug 1797755 - Part 4: Remove option to set maximum mark stack capacity in release builds r=sfink
#   
#   Currently we have a GC parameter that allows setting a maximum mark stack
#   capacity. This is only ever used by test code, not in the browser. This
#   requires extra unnecessary work in release builds if we move to a different
# ==============================================================================

diff -r 6e5a935e576a -r 098569654f17 js/public/GCAPI.h
--- a/js/public/GCAPI.h	Fri Oct 28 09:47:14 2022 +0000
+++ b/js/public/GCAPI.h	Fri Oct 28 09:47:14 2022 +0000
@@ -130,14 +130,6 @@
   JSGC_SLICE_TIME_BUDGET_MS = 9,
 
   /**
-   * Maximum size the GC mark stack can grow to.
-   *
-   * Pref: none
-   * Default: MarkStack::DefaultCapacity
-   */
-  JSGC_MARK_STACK_LIMIT = 10,
-
-  /**
    * The "do we collect?" decision depends on various parameters and can be
    * summarised as:
    *